server/pages/components/Layout.js

```diff
@@ -6,6 +6,11 @@ import Analytics from '../../../shared/components/Analytics';
 import globalStyles from '../../../shared/styles/globals';
 import { color } from '../../../shared/styles/constants';
 
+export const title = 'Outline';
+export const description =
+  'Your team’s knowledge base - Team wiki, documentation, playbooks, onboarding & more…';
+export const screenshotUrl = 'https://www.getoutline.com/screenshot.png';
```

Comment:
There's a env var for URL which is set to `https://www.getoutline.com` in prod, we should use that instead of hardcoding here. Otherwise 👍

Reply:
We should probably take these out all together for non-SaaS installation

Reply:
Changed. I added robots exclusion for non-hosted variant